pkg/ginutils/gincache/cache.go

```diff
@@ -120,16 +119,12 @@ func (mc *memCache) getCache(key string) (*cacheResponse, error) {
 	if !ok {
 		return nil, fmt.Errorf("gin-cache: invalid cache key %s", key)
 	}
-	r, ok := c.([]byte)
+	r, ok := c.(cacheResponse)
```

Comment:
the cache is saved in a byte array

https://github.com/trustwallet/blockatlas/pull/813/files#diff-70e12b2752974d408c475deba3e9a7a5R107

Reply:
Try to debug this line. Cast always fails since `c` comes as `cacheResponse` struct